packages/block-library/src/post-comment-author/edit.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * WordPress dependencies
+ */
+import ServerSideRender from '@wordpress/server-side-render';
+
+// TODO: JSDOC types
+export default function Edit( { attributes, context } ) {
+	const { className } = attributes;
+	const { commentId } = context;
+
+	return (
+		<p className={ className }>
+			<ServerSideRender
+				block="core/post-comment-author"
+				attributes={ { commentId } }
+			/>
```

Comment:
As discussed on Slack, let's preferably use client-side rendered stuff :slightly_smiling_face:

Reply:
Specifically, would

```
useEntityProp(
	'root',
	'comment',
	'author',
	commentId
)
```

work?

Reply:
Yes, I already had the ServerSideRenderer implemented so pushed that before signing off. Replaced ServerSideRenderer with client side implementation.